frame/staking/src/lib.rs

```diff
@@ -696,21 +702,45 @@ pub struct IndividualExposure<AccountId, Balance: HasCompact> {
 }
 
 /// A snapshot of the stake backing a single validator in the system.
-#[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
-pub struct Exposure<AccountId, Balance: HasCompact> {
+#[derive(
+	PartialEqNoBound,
+	EqNoBound,
+	PartialOrd,
+	Ord,
+	CloneNoBound,
+	Encode,
+	Decode,
+	RuntimeDebugNoBound,
+	TypeInfo,
+	MaxEncodedLen,
+)]
+#[codec(mel_bound())]
+#[scale_info(skip_type_params(MaxNominations))]
+pub struct Exposure<AccountId, Balance, MaxNominations>
+where
+	AccountId: Debug + Encode + Decode + MaxEncodedLen + Clone + Eq + PartialEq,
+	Balance: Debug + HasCompact + Encode + Decode + MaxEncodedLen + Clone + Eq + PartialEq,
+	MaxNominations: Get<u32>,
```

Comment:
Not a good name -- we use this term to determine the number of nominations that a single nominator can have. i.e. 16. 

I suggest using `MaxBackers` here instead.